docker/README.md

```diff
@@ -0,0 +1,122 @@
+
+### Docker Compose
+
+You can also run the `txsim` Docker image using Docker Compose. Here's an example `docker-compose.yml` file:
+
+```yaml
+version: '3'
+services:
+  txsim:
+    image: ghcr.io/celestiaorg/txsim
+    command: >
+      -k 0
+      -r http://consensus-validator-robusta-rc6.celestia-robusta.com:26657,http://consensus-full-robusta-rc6.celestia-robusta.com:26657
+      -g consensus-validator-robusta-rc6.celestia-robusta.com:9090
+      -t 10s
+      -b 10
+      -d 100
+      -e 10
+    volumes:
+      - /Users/txsimp/.celestia-app:/home/celestia
+```
+
+In this file, the `volumes` key is used to mount the `/Users/txsimp/.celestia-app` directory from the host to the `/home/celestia` directory in the Docker container.
+
+### Kubernetes Deployment
+
+Finally, you can run the `txsim` Docker image in a Kubernetes cluster. Here's an example `deployment.yaml` file:
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: txsim-deployment
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: txsim
+  template:
+    metadata:
+      labels:
+        app: txsim
+    spec:
+      containers:
+      - name: txsim
+        image: ghcr.io/celestiaorg/txsim
+        args:
+        - "-k"
+        - "0"
+        - "-r"
+        - "http://consensus-validator-robusta-rc6.celestia-robusta.com:26657,http://consensus-full-robusta-rc6.celestia-robusta.com:26657"
+        - "-g"
+        - "consensus-validator-robusta-rc6.celestia-robusta.com:9090"
+        - "-t"
+        - "10s"
+        - "-b"
+        - "10"
+        - "-d"
+        - "100"
+        - "-e"
+        - "10"
+        volumeMounts:
+        - name: keyring-volume
+          mountPath: /home/celestia
+      volumes:
+      - name: keyring-volume
+        hostPath:
+          path: /Users/txsimp/.celestia-app
+```
+
+In this file, the `volumeMounts` and `volumes` keys are used to mount the `/Users/txsimp/.celestia-app` directory from the host to the `/home/celestia` directory in the Docker container.
+
+## Flag Breakdown
+
+Here's a breakdown of what each flag means:
+
+- `-k`: Whether a new key should be created (1 for yes, 0 for no)
+- `-p`: The path to the keyring for the prefunded account
+- `-r`: The RPC endpoints for the `txsim` binary
+- `-g`: The gRPC endpoints for the `txsim` binary
+- `-t`: The poll time for the `txsim` binary
+- `-b`: The number of blob sequences to run
+- `-a`: The range of blobs to send per PFB in a sequence
+- `-s`: The range of blob sizes to send
+- `-m`: The mnemonic for the keyring
+- `-d`: The seed for the random number generator
+- `-e`: The number of send sequences to run
+- `-i`: The amount to send from one account to another
+- `-v`: The number of send iterations to run per sequence
+- `-u`: The number of stake sequences to run
+- `-w`: The amount of initial stake per sequence
```

Comment:
ahh this legend really helps. It may be even clearer to use longer format flags. I.e. `--seed`, `--key`, etc. but not blocking